sequence_processing_pipeline/Job.py

```diff
@@ -47,6 +47,12 @@ def __init__(self, root_dir, output_path, job_name, executable_paths,
         # the version given, raise a PipelineError.
         for executable_path in executable_paths:
             file_path, file_name = split(executable_path)
+
+            # bcl-convert module is not installed on the node this test gets
+            # run on, so forego it entirely.
+            if 'bcl-convert' in file_name:
```

Comment:
This is not OK for deployment correct? If I'm wrong, could you add a better explanation of why this is OK?

Reply:
No problem. Just to record it here as well: This code is running on qiita-rc which doesn't have access to the bcl-convert module for reasons Jeff can explain. However it is available on brncl-n barnacle nodes, which actually run bcl-convert. If this bypass is not here, the pipeline will falsely determine bcl-convert isn't available and exit with an error.

Since this is a hardcode, perhaps it would better to generalize it.